Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -1351,3 +1351,15 @@ void MainWindow::OnUpdateProgressDialog(QString title, int progress, int total)
     m_progress_dialog = nullptr;
   }
 }
+
+void MainWindow::show()
```

Comment:
Is having a method with the same name as a non-virtual method in the base class a good idea?

Reply:
If you've got a better idea feel free to suggest something else.

Reply:
Since it's not overriding any Qt function you should at least call this `Show` to follow the naming conventions and to make it clear that show() is not virtual. Otherwise it's kind of misleading.